### PR TITLE
add arguments for plot_wigner_fock_distribution

### PR DESCRIPTION
**Pull request by KosukeMizuno**
**Description**

Related to #1112. The scalling of Winger fuction can be changed in `wigner()` (default is `g=sqrt(2)`). But it cannot be specified with `plot_wigner()` or `plot_wigner_fock_distribution()`. For my usage, it should be `g=2`, so it was inconvenient. I modified these two plotter functions to pass `g` (and the other args of `wigner()`).

### CODE REVIEW

File: qutip/visualization.py [hunk 1]
```diff
@@ -1162,7 +1175,7 @@ def plot_wigner(rho, xvec=None, yvec=None, method='clenshaw',
 
     fig : a matplotlib Figure instance, optional
         The Figure canvas in which the plot will be drawn.
-
+
```
Comment (Ericgig) [inline qutip/visualization.py]:
```suggestion
```

File: qutip/visualization.py [hunk 2]
```diff
@@ -1180,6 +1193,7 @@ def plot_wigner(rho, xvec=None, yvec=None, method='clenshaw',
 
     if not isinstance(rho, list):
         rhos = [rho]
+
```
Comment (Ericgig) [inline qutip/visualization.py]:
```suggestion
```

**Review by Ericgig — APPROVED** [review]
Sorry @KosukeMizuno for letting this fall though the cracks.
The change looks good, thank you for the contribution.

**Comment by Ericgig** [comment]
Since this PR was made, a new function, `anim_wigner`, was added to make animations using `plot_wigner`. 
Changing `plot_wigner` signature created a conflict that was not visible using github conflict resolution tool...
@KosukeMizuno would you be able to fix that conflict?

**Comment by KosukeMizuno** [comment]
oh you were much faster than me. Anyway thanks for approving this change :)

**Pull request merged**